src/obstacle_detector.cpp

```diff
@@ -378,6 +482,44 @@ float ObstacleDetector::obstacle_angle(bool left)
            }
         }
     }
+
+    // Draw rotated footprint to show limit of rotation
```

Comment:
Does it really make sense to draw the whole footprint?

I think we should be only drawing the side of the box in the direction of the turn. So if turning left, only show the left side of the box.

Reply:
When turning, an obstacle collision could be with any of the four sides of the robot.

Reply:
![rotate](https://user-images.githubusercontent.com/5511255/35486018-c4c4cb28-041c-11e8-845f-2e42c675da80.png)

Here's an obstacle in front